src/components/album/Album.js

```diff
@@ -53,23 +57,40 @@ function Album({ album, wishListId }) {
         <AddToWishlistButton album={album} wishListId={wishListId} />
       </ButtonGroup>
       {/* Dialog */}
-      <Dialog open={openDialog} onClose={handleCloseDialog}>
+      <Dialog
+        open={openDialog}
+        onClose={handleCloseDialog}
+        fullWidth={fullWidth}
+        maxWidth={maxWidth}
+      >
         <DialogActions>
           <Button onClick={handleCloseDialog} color="primary">
             X
           </Button>
         </DialogActions>
         <DialogContent>
-          <DialogContentText>
+          <DialogContentText
+            style={{
+              display: 'flex',
+              flexDirection: 'column',
+              justifyContent: 'center',
+            }}
+          >
             <div>
               <div>
                 Release Date: {new Date(album.releaseDate).toLocaleDateString()}
               </div>
               <div>Average Rating: {album.averageRating}</div>
             </div>
           </DialogContentText>
-          <AlbumPreview spotifyUrl={album.spotifyUrl} />
         </DialogContent>
+        <AlbumPreview>
+          <AlbumReviews
+            albumId={album._id}
+            spotifyUrl={album.spotifyUrl}
+            numOfReviews={album.numOfReviews}
```

Comment:
`AlbumReviews.js` only shows the component accepting one prop: `albumId`. Are these other two (`spotifyUrl` and `numOfReviews` expected to be used by the `AlbumReviews` component for anything?

Reply:
@akosasante they don't.. will delete them... to avoid confusion I realize I should have named AlbumReviews component as ReviewsList.js as the file's name,..